commit 5b2cab42ece4dc10d526d7c535811f1dcaaf6401 (refs/changes/22/4722/4)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-08-20T21:31:53+00:00 (1 year, 2 months ago)
    
    OS-7148 Joyent-M12G5 doesn't match shipping configuration
    Reviewed by: Rob Johnston <rob.johnston@joyent.com>
    Approved by: Rob Johnston <rob.johnston@joyent.com>

diff --git a/usr/src/lib/fm/topo/maps/SMCI,SSG-2029P-ACR24L/SSG-2029P-ACR24L-disk-hc-topology.xmlgenksh b/usr/src/lib/fm/topo/maps/SMCI,SSG-2029P-ACR24L/SSG-2029P-ACR24L-disk-hc-topology.xmlgenksh
index 9ab078cff4..327c41c53b 100755
--- a/usr/src/lib/fm/topo/maps/SMCI,SSG-2029P-ACR24L/SSG-2029P-ACR24L-disk-hc-topology.xmlgenksh
+++ b/usr/src/lib/fm/topo/maps/SMCI,SSG-2029P-ACR24L/SSG-2029P-ACR24L-disk-hc-topology.xmlgenksh
@@ -106,7 +106,7 @@ EOF
 enclosure=1
 bay=0
 slot=0
-devctl0='/devices/pci@7c,0/pci8086,2030@0/pci15d9,808@0:devctl'
+devctl0='/devices/pci@7d,0/pci8086,2030@0/pci15d9,808@0:devctl'
 while (( slot <= 7 )); do
   do_sas_node $bay "Front Disk $bay" "$devctl0" $enclosure $slot
   (( bay = bay + 1 ))
@@ -114,7 +114,7 @@ while (( slot <= 7 )); do
 done
 
 slot=0
-devctl0='/devices/pci@cc,0/pci8086,2030@0/pci15d9,808@0:devctl'
+devctl0='/devices/pci@cd,0/pci8086,2030@0/pci15d9,808@0:devctl'
 while (( slot <= 7 )); do
   do_sas_node $bay "Front Disk $bay" "$devctl0" $enclosure $slot
   (( bay = bay + 1 ))
@@ -122,7 +122,7 @@ while (( slot <= 7 )); do
 done
 
 slot=0
-devctl0='/devices/pci@55,0/pci8086,2030@0/pci15d9,808@0:devctl'
+devctl0='/devices/pci@56,0/pci8086,2030@0/pci15d9,808@0:devctl'
 while (( slot <= 7 )); do
   do_sas_node $bay "Front Disk $bay" "$devctl0" $enclosure $slot
   (( bay = bay + 1 ))
